### PR TITLE
Add jc.nvim

### PR DESCRIPTION
**Pull request by zjp-CN**

### CODE REVIEW

File: contents/2022/Oct/10.md [hunk 1]
```diff
@@ -268,6 +268,38 @@ A great new plugin!
...
+        <span class="icon-text">
+            <span class="icon">
+                <i class="fa-solid fa-book"></i>
+            </span>
+            <span>jc.nvim</span>
+        </span>
+    </a>
+</h3>
+
+jc.nvim – successor of [vim-javacomplete2](https://github.com/artur-shaik/vim-javacomplete2)
```
Comment (phaazon) [inline contents/2022/Oct/10.md:282]:
```suggestion
`jc.nvim` – successor of [vim-javacomplete2](https://github.com/artur-shaik/vim-javacomplete2)
```

File: contents/2022/Oct/10.md [hunk 2]
```diff
@@ -268,6 +268,38 @@ A great new plugin!
...
+which use neovim's built-in LSP client and [Eclipse JDT Language Server](https://github.com/eclipse/eclipse.jdt.ls).
```
Comment (phaazon) [inline contents/2022/Oct/10.md:283]:
```suggestion
which uses Neovim's built-in LSP client and [Eclipse JDT Language Server](https://github.com/eclipse/eclipse.jdt.ls).
```

File: contents/2022/Oct/10.md [hunk 3]
```diff
@@ -268,6 +268,38 @@ A great new plugin!
...
+
+Main goal of this project is to migrate functionallty of jc2.
```
Comment (phaazon) [inline contents/2022/Oct/10.md:285]:
```suggestion
Main goal of this project is to migrate all the functionallties of jc2 to Neovim.
```

File: contents/2022/Oct/10.md [hunk 4]
```diff
@@ -268,6 +268,38 @@ A great new plugin!
...
+
+In addition to autocompletion it can:
```
Comment (phaazon) [inline contents/2022/Oct/10.md:287]:
```suggestion
In addition to autocompletion, it can:
```

File: contents/2022/Oct/10.md [hunk 5]
```diff
@@ -268,6 +268,38 @@ A great new plugin!
...
+
+- organize imports;
+- generate code (`toString`, `hashCode`, `equals`, constructors, accessors) with field selection;
+- add abstract methods to implementing class;
+- execute [vimspector](https://github.com/puremourning/vimspector) debug session;
+- automatic installation of jdt.ls and java-debug extension;
+- class creation methods from `jc2`.
+
+It's written by [@artur-shaik](https://github.com/artur-shaik) and you read his [blog post](https://shaik.link/posts/javacomplete-to-jc.nvim/) about it.
```
Comment (phaazon) [inline contents/2022/Oct/10.md:296]:
```suggestion
It's written by [@artur-shaik](https://github.com/artur-shaik) and you can read his [blog post](https://shaik.link/posts/javacomplete-to-jc.nvim/) about it.
```

**Review by phaazon — CHANGES_REQUESTED** [review]
A couple of nits but overall okay, thank you!

**Comment by zjp-CN** [comment]
Seems I should submit this PR to [2022-Oct-17](https://github.com/phaazon/this-week-in-neovim-contents/tree/2022-Oct-17) branch? I realized the branch name indicates the deadline on merge...

**Comment by phaazon** [comment]
> Seems I should submit this PR to [2022-Oct-17](https://github.com/phaazon/this-week-in-neovim-contents/tree/2022-Oct-17) branch? I realized the branch name indicates the deadline on merge...

Yes, please change the base branch. You shouldn’t have any issues.

**Pull request closed without merging**